Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -137,6 +160,12 @@ private PropertyInfo FindProperty(string readName)
             var name = options.NamingConvention.ToProperty(readName);
             return targetType.GetProperty(name);
         }
+
+        private IEnumerable<PropertyInfo> ReadOnlyProperties(Type targetType)
+        {
+            return targetType.GetProperties().Where(p => !p.CanWrite);
```

Comment:
This does not define a readonly property, this is simply a property with no setter. See this link for a description of how to find readonly properties: https://stackoverflow.com/questions/5243764/at-runtime-how-can-i-test-whether-a-property-is-readonly.

Reply:
Addressed.